ui/src/testing/stubs/user-api-stub.ts

```diff
@@ -8,14 +8,10 @@ import {
   WorkbenchListBillingAccountsResponse,
 } from 'generated/fetch';
 
-import { stubNotImplementedError } from 'testing/stubs/stub-utils';
-
 export class UserApiStub extends UserApi {
   existingUsers: UserRole[];
   constructor(existingUsers?: UserRole[]) {
-    super(undefined, undefined, (..._: any[]) => {
-      throw stubNotImplementedError;
-    });
+    super(undefined);
```

Comment:
This ultimately extends BaseApi which only takes one parameter: a configuration object.